packages/atomic/cypress/integration/facets/category-facet/category-facet-assertions.ts

```diff
@@ -66,12 +63,15 @@ export function assertLogFacetSelect(path: string[]) {
         hierarchicalField
       );
 
-      expect(analyticsBody.facetState[0]).to.have.property('state', 'selected');
-      expect(analyticsBody.facetState[0]).to.have.property(
+      expect(analyticsBody.facetState?.[0]).to.have.property(
```

Comment:
A side effect of `analyticsBody` now being typed is that I needed to make small adjustments like this one.